server/graphql/v2/mutation/VirtualCardMutations.ts

```diff
@@ -67,6 +67,7 @@ const virtualCardMutations = {
         cardNumber,
         expireDate,
         cvv,
+        args.virtualCard.name,
```

Comment:
Interesting that it was already available in `VirtualCardInput`

Reply:
I would have say "weird" ^^